docker/bigdl-k8s/Dockerfile

```diff
@@ -20,16 +24,16 @@ RUN apt-get update --fix-missing && \
     apt-get install -y apt-utils vim curl nano wget unzip maven git && \
 # java
     wget $JDK_URL && \
```

Comment:
can we wget  a file and save as a unified filename
and use this unified name to continue followed commands:
```
wget -O jdk.tar.gz $JDK_URL
gunzip jdk.tar.gz && \
tar -xf jdk.tar --directory /opt/jdk$JDK_VERSION --strip-components 1 && \
rm jdk.tar && \
```